XAMLator.Core.Net461/XAMLator.Core.Net461.csproj

```diff
@@ -0,0 +1,54 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same as for the XAMLator.Client, is there is a reason why this has to be converted into a library instead of shared project?